public/app/features/alerting/unified/components/silences/SilencesEditor.tsx

```diff
@@ -164,6 +164,7 @@ export const SilencesEditor: FC<Props> = ({ silence, alertManagerSourceName }) =
     700,
     [clearErrors, duration, endsAt, prevDuration, setValue, startsAt]
   );
+  const userLogged = Boolean(config.bootData.user.isSignedIn && config.bootData.user.name);
```

Comment:
It should work only for anonymous users, I think
```suggestion
  const userLogged = Boolean(config.bootData.user.isSignedIn && config.bootData.user.name);
```

Reply:
opsssss, trueee

Reply:
My bad, I haven't noticed `!` 🤦

Reply:
we use the userLogged exactly for this: 
`         {!userLogged && (
            <Field
              className={cx(styles.field, styles.createdBy)}
              label="Created By"
              required`